### PR TITLE
FeatureAgglomeration addition 

### PR DESCRIPTION
**Pull request by AHReccese**
#### What does this implement/fix? Explain your changes.
add support for FeatureAgglomeration model
add basic function transporter

### CODE REVIEW

File: tests/test_clusterings/test_clusterings.py [hunk 1]
```diff
@@ -25,7 +26,7 @@
     "OPTICS": [optics],
     "SPECTRAL_CLUSTERING": [spectral_clustering],
     "GAUSSIAN_MIXTURE": [gaussian_mixture],
-    "HIERARCHICAL_CLUSTERING":[agglomerative_clustering],
+    "HIERARCHICAL_CLUSTERING":[agglomerative_clustering, feature_agglomeration],
```
Comment (sadrasabouri) [inline tests/test_clusterings/test_clusterings.py:29]: `"HIERARCHICAL_CLUSTERING": [agglomerative_clustering, feature_agglomeration],`

File: pymilo/utils/util.py [hunk 2]
```diff
@@ -89,3 +90,19 @@ def all_same(arr):
     :return: bool
     """
     return all(x == arr[0] for x in arr)
+
+
+def import_function(module_name, function_name):
+    """
+    Import function with name function_name from module called module_name.
+
+    :param module_name: module to import associated function from
```
Comment (sadrasabouri) [inline pymilo/utils/util.py:99]: `module to import the function from`

File: pymilo/utils/util.py [hunk 3]
```diff
@@ -89,3 +90,19 @@ def all_same(arr):
...
+    :type module_name: str
+    :param function_name: associated function's name to get imported
```
Comment (sadrasabouri) [inline pymilo/utils/util.py:101]: `function's name to get imported`

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Looks great. I left minor comments.

**Comment by AHReccese** [comment]
I applied all of your comments, any other suggestion? 
@sadrasabouri

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- `import_function` method in `util.py`
+- `Function` Transporter
+- `FeatureAgglomeration ` model
```
Comment (sadrasabouri) [inline CHANGELOG.md:11]: One extra space is inserted here.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**